cmd/mms/commands.go

```diff
@@ -138,3 +146,39 @@ func productReceiver(event *mms.ProductEvent) error {
 	fmt.Println(event)
 	return nil
 }
+
+func createExecutableCallback(filepath string, filter bool) func(event *mms.ProductEvent) error {
+	_, err := exec.LookPath(filepath)
+	fmt.Println(filepath)
+	if err != nil {
+		panic(fmt.Errorf("command executable not found, %s", err.Error()))
```

Comment:
Could this be some less fatal info to the user that the executable is not found? There is no reason to panic because of that I think.

Reply:
This part is only run if command is given, so not sure if non-fatal is needed? Either we need to restructure the callbackcreators to return errors or just print the error

Reply:
Fixed by 57df8a1 and 7821957